src/ImageSharp.Web/Caching/PhysicalFileSystemCache.cs

```diff
@@ -54,18 +48,13 @@ public class PhysicalFileSystemCache : IImageCache
             FormatUtilities formatUtilities)
         {
             Guard.NotNull(environment, nameof(environment));
-            Guard.NotNull(options, nameof(options));
```

Comment:
We still need the null check for options.

Reply:
I removed this check, because the code below did a conditional null check:

https://github.com/SixLabors/ImageSharp.Web/blob/f97516e3a3461c2e3c0a3d10236de70e1bcaa63b/src/ImageSharp.Web/Caching/PhysicalFileSystemCache.cs#L53-L54

The options will always be registered when using `builder.AddImageSharp()` though, so I've added the guard again and removed this custom logic...

Reply:
Ah.... Actually I'm not sure about that. I can't see where the options would automatically be created.

Reply:
Looks like registration isn't needed anyway: https://stackoverflow.com/questions/55074806/why-ioptions-is-getting-resolved-even-if-not-registered

Reply:
TIL !!